move_backwards_recovery/src/move_backwards_recovery.cpp

```diff
@@ -115,24 +115,6 @@ void MoveBackRecovery::runBehavior(){
   }
   runOnce_ = true;
 
```

Comment:
need to compare this to the implementation in mfp_executive to make sure the logic is the same

Reply:
I went through, and it seems like it does